controllers/handlers/wasp-agent/scc.go

```diff
@@ -0,0 +1,76 @@
+package wasp_agent
+
+import (
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/kubevirt/hyperconverged-cluster-operator/controllers/operands"
+
+	securityv1 "github.com/openshift/api/security/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	hcov1beta1 "github.com/kubevirt/hyperconverged-cluster-operator/api/v1beta1"
+)
+
+func NewSecurityContextConstraintHandler(Client client.Client, Scheme *runtime.Scheme) operands.Operand {
```

Comment:
nit: Please use a specific name. We can use `SCC` to shorten it; e.g.

`NewWaspAgentSCCHandler`